tests/test_shipitapi.py

```diff
@@ -0,0 +1,193 @@
+import datetime
+import json
+import mock
+import pytest
+import redo
+import requests
+
+
+# mock redo library, to make test faster
+real_retry = redo.retry
+
+
+def fake(*args, **kwargs):
+	kwargs.pop('sleeptime')
+	kwargs.pop('max_sleeptime')
+	return real_retry(sleeptime=1, max_sleeptime=1, jitter=1, *args, **kwargs)
+
+
+redo.retry = fake
+
+
+def test_simple():
+	assert 1 + 1 == 2
+
+
+def create_token(expiry): # expiry is in UTC
+	"""
+	csrf token format is %Y%m%d%H%M%S##...
+	"""
+	return expiry.strftime('%Y%m%d%H%M%S') + '##....'
+
+
+def test_csrf_token_expired(mocker):
+	from src.shipitapi.shipitapi import is_csrf_token_expired
+
+	now = datetime.datetime.utcnow()
```

Comment:
This looks good, but just FYI - playing around with time can also be done via the `freezegun` lib. There's an example on how to use this in [this](https://github.com/mozilla-releng/shipitscript/blob/master/shipitscript/test/integration/test_integration_script.py#L49) file.

Reply:
I see! I'll just change it to use freezegun, since it looks cleaner, thanks!